benchmark/sql_regressions/run_regressions.sh

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+set -eo pipefail
+
+function fail() {
+    1>&2 echo "$@"
+    exit 1
+}
+
+logictest="../../go/libraries/doltcore/sqle/logictest"
+logictest_main="$logictest"/main
+
+if [[ "$#" -ne 1 ]]; then
+    fail Usage: ./run_regressions.sh ENV_VARIABLES_FILE
+fi
+
+source "$1"
+if [ -z "$DOLT_ROOT_PATH" ]; then fail Must supply DOLT_ROOT_PATH; fi
+if [ -z "$DOLT_CONFIG_PATH" ]; then fail Must supply DOLT_CONFIG_PATH; fi
+if [ -z "$DOLT_GLOBAL_CONFIG" ]; then fail Must supply DOLT_GLOBAL_CONFIG; fi
+if [ -z "$CREDSDIR" ]; then fail Must supply CREDSDIR; fi
+if [ -z "$DOLT_CREDS" ]; then fail Must supply DOLT_CREDS; fi
+if [ -z "$CREDS_HASH" ]; then fail Must supply CREDS_HASH; fi
+if [ -z "$JOB_TYPE" ]; then fail Must supply DOLT_VERSION; fi
+if [ -z "$TEST_N_TIMES" ]; then fail Must supply DOLT_VERSION; fi
+if [ -z "$FAIL_ON_EXISTING_VERSION" ]; then fail Must supply FAIL_ON_EXISTING_VERSION; fi
+
+if [[ -z "$DOLT_VERSION" ]] && [[ -z "$DOLT_RELEASE" ]]; then
+  fail Must supply DOLT_VERSION;
+  elif [[ -z "$DOLT_VERSION" && -n "$DOLT_RELEASE" ]]; then
```

Comment:
nit: indent this at column 0

Reply:
nit: here we do `[[ ... && ... ]]` and above we do `[[ ... ]] && [[ ... ]]`. Is there a difference in semantics? Should be consistent between the two?